spec/operators/single-spec.ts

```diff
@@ -1,173 +1,331 @@
 import { expect } from 'chai';
-import { hot, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
 import { single, mergeMap, tap } from 'rxjs/operators';
-import { of, EmptyError } from 'rxjs';
-
-declare function asDiagram(arg: string): Function;
```

Comment:
IDK what the consequences of removing this `asDiagram` business are? Will it break website images when the docs are re-created?

Reply:
From what I remember, we stop generating them like this, and we're just using old, static images. @JWO719, can you confirm this? I've been removing these `asDiagram` calls when I see them.

Reply:
Lol I never noticed. So it's kinda true, that we are using a static old version of the generated marbleDiagrams, but there's a pr and several issues to improve this mechanism and IIRC it's still using this  `asDiagram` function, so it would be good to keep it for now. I can go through the history to find the deleted images.

This explains now why there is always a diff when I try to generate the new images :D

Reply:
@niklas-wortmann ... can you please set up a separate issue? I've been removing these for the last year at least, because at one point we stopped using them. It's always the first test in the file that had it before. It would be nice if this was something that was imported and used, rather than a global variable, too.